### PR TITLE
[FIX] inventory code comments

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: inventory_api/src/main/java/com/intel/dai/inventory/api/ForeignHwInventoryRequester.java [hunk 1]
```diff
@@ -53,6 +63,10 @@ public int getDiscoveryStatus() {
         }
         return 1;
     }
+
+    /**
+     * This method is used to get teh hardware inventory data.
```
Comment (nbwhite) [inline inventory_api/src/main/java/com/intel/dai/inventory/api/ForeignHwInventoryRequester.java]: "Used to get teh", should be the, present in several other locations as well.

File: inventory/src/main/java/com/intel/dai/inventory/ForeignSubscriptionResponseParser.java [hunk 2]
```diff
@@ -18,6 +18,9 @@
 public class ForeignSubscriptionResponseParser implements SubscriptionResponseParser {
     public ForeignSubscriptionResponseParser() {}
 
+    /**
+     * This method is sed to parser response receiver.
```
Comment (avincigu) [inline inventory/src/main/java/com/intel/dai/inventory/ForeignSubscriptionResponseParser.java]: Typo: I think sed means used

File: inventory_api/src/main/java/com/intel/dai/inventory/api/ForeignHwInventoryRequester.java [hunk 3]
```diff
@@ -67,6 +81,10 @@ public int getDiscoveryStatus() {
         }
         return new ImmutablePair<>(1, "");
     }
+
+    /**
+     * This method is used to get teh hardware inventory datafor a location.
```
Comment (avincigu) [inline inventory_api/src/main/java/com/intel/dai/inventory/api/ForeignHwInventoryRequester.java]: Same typo here as the one Nathan pointed out above

File: inventory_api/src/main/java/com/intel/dai/inventory/api/HWInvDiscovery.java [hunk 4]
```diff
@@ -20,29 +20,45 @@
...
     public HWInvDiscovery(Logger logger) {
         log = logger;
 
         GsonBuilder builder = new GsonBuilder();
         builder.setPrettyPrinting();
         gson = builder.create();
     }
+
+    /**
+     * This method is used initialise rest client.
```
Comment (avincigu) [inline inventory_api/src/main/java/com/intel/dai/inventory/api/HWInvDiscovery.java]: I think it is initialize

File: inventory_api/src/main/java/com/intel/dai/inventory/api/HWInvDiscovery.java [hunk 5]
```diff
@@ -51,6 +67,9 @@ public int pollForDiscoveryProgress() {
         return requester_.getHwInventory(foreignName);
     }
 
+    /**
+     * This method is used to get teh hardware inventory data for all locations.
```
Comment (avincigu) [inline inventory_api/src/main/java/com/intel/dai/inventory/api/HWInvDiscovery.java]: the

File: inventory_api/src/main/java/com/intel/dai/inventory/api/HWInvDiscovery.java [hunk 6]
```diff
@@ -59,6 +78,9 @@ public int pollForDiscoveryProgress() {
         return requester_.getHwInventory();
     }
 
+    /**
+     * This method is used to create/initialise client.
```
Comment (avincigu) [inline inventory_api/src/main/java/com/intel/dai/inventory/api/HWInvDiscovery.java]: initialize

**Comment by sindhu-106** [comment]
changes are done.

File: inventory_api/src/main/java/com/intel/dai/inventory/api/ForeignHwInventoryRequester.java (hunk 1 above)
Comment (sindhu-106) [reply inventory_api/src/main/java/com/intel/dai/inventory/api/ForeignHwInventoryRequester.java]: changes are done.

**Pull request merged**